src/header/mod.rs

```diff
@@ -546,6 +551,32 @@ impl fmt::Debug for Headers {
     }
 }
 
+#[cfg(feature = "compat")]
+impl From<http_types::HeaderMap> for Headers {
+    fn from(header_map: http_types::HeaderMap) -> Headers {
+        let mut headers = Headers::new();
+        for (name, value) in header_map.iter() {
+            headers.set_raw(name.as_str().to_string(), value.as_bytes());        
```

Comment:
This should use `append_raw`, since `set_raw` will overwrite any previous value for a header name.

Reply:
You got it!